src/main/kotlin/com/bridgecrew/ui/buttons/SuppressionButton.kt

```diff
@@ -37,17 +39,24 @@ class SuppressionButton(private var result: BaseCheckovResult): CheckovLinkButto
         }
 
         val dialog = SuppressionDialog()
-        dialog.show()
-        if(dialog.exitCode == DialogWrapper.OK_EXIT_CODE) {
-            generateComment(fileType, dialog.userJustification)
+        if(isOpenDialog) {
```

Comment:
The name is a little confusing, perhaps:
`shouldOpenDialog` or `!isDialogOpened` ?

Reply:
it's before opening it, so it's the action of allowing it to be opened